### ISSUE TITLE
Purge our project of the term 'expire'.

### ISSUE THREAD

**Issue by paytonshaltis** [sderacy/gooncard#153]
With the new way we are implementing dynamic cards, correct me if I'm wrong but I believe that we scrapped the expiration functionality? If so, there are a few screens that show it (notfound page for example) that probably need to drop the term.

**Comment by sderacy** [comment]
If I understand how the cards are generated, a user can overwrite a previous card with their new card right? If this is done, what do you think the notfound page should say to the user to indicate that the card no longer exists?

**Comment by paytonshaltis** [comment]
> If I understand how the cards are generated, a user can overwrite a previous card with their new card right? If this is done, what do you think the notfound page should say to the user to indicate that the card no longer exists?

So anytime you click generate, a brand new instance of the card is created. The only time a card would cease to exist (in the current implementation) is if all of the accounts included in that card were deleted by the user, or if the user deletes their account. These cases are already handled, and the user is redirected to the not found page.

Honestly, I think @lelekaz 's fix in #154 takes care of this just fine; I don't think we have it mentioned anywhere else.

**Comment by sderacy** [comment]
>So anytime you click generate, a brand new instance of the card is created.
I may have been mistaken but I thought with the explanation you provide earlier in the semester, there's essentially one card that is associated with each user. It sounds like you're saying one user could effectively make 30 cards if they wanted to.

>These cases are already handled, and the user is redirected to the not found page.

Honestly, I think @lelekaz 's fix in https://github.com/sderacy/gooncard/pull/154 takes care of this just fine; I don't think we have it mentioned anywhere else.

**This is incredibly nitpicky** but even if we don't explicitly use the term "expire", I think the page should at least explain how this error could've happened. Adding "you can no longer see this because the user deleted their account" or something similar would've been fine in my opinion. If we think that saying "this card is nonexistent" encompasses both the users trying to access a now invalid card and a url that was never generated, then it's fine.

**Comment by paytonshaltis** [comment]
> I may have been mistaken but I thought with the explanation you provide earlier in the semester, there's essentially one card that is associated with each user. It sounds like you're saying one user could effectively make 30 cards if they wanted to.

That's correct, they could. If I want to give you my Instagram and email but I want to give Leah just my Instagram, clicking generate for each of you essentially makes two entries for Instagram and one for email, each with their associated UUID. In effect, there are no 'cards', but there are sets of accounts that can be accessed via a UUID, giving the _representation_ of a card. 

When the user goes to retrieve a card via the link, they are simply getting all of the accounts associated with some UUID. If I delete my Instagram, you'll still be able to see my email, but Leah will get the error page because all of the accounts for the card I generated for her have been deleted. It may be easier to demo this to you in person if you want.

> --**This is incredibly nitpicky**-- but even if we don't explicitly use the term "expire", I think the page should at least explain how this error could've happened. Adding "you can no longer see this because the user deleted their account" or something similar would've been fine in my opinion. If we think that saying "this card is nonexistent" encompasses both the users trying to access a now invalid card and a url that was never generated, then it's fine.

Unfortunately, there's no easy way to make the distinction. If the UUID is not in the database, we cannot determine if it was deleted or never there (without of course permanently storing all of the cards that ever existed). I do like the idea of a better explanation: rather than just saying it doesn't exist, we can tell the user that either (a) all of the accounts associated with that card were deleted by the owner, or (b) the link is invalid. That's about the best we can do without increasing complexity.

**Comment by sderacy** [comment]
>we can tell the user that either (a) all of the accounts associated with that card were deleted by the owner, or (b) the link is invalid

Yup, I don't want to touch anything back end wise for this. I'm suggesting this, which would just be a changing of text in the not found page

**Comment by paytonshaltis** [comment]
That sounds like an A1 plan partner.
